client/state/themes/selectors.js

```diff
@@ -96,6 +96,17 @@ export const getThemesForQuery = createSelector(
 );
 
 /**
+ * Returns last query used.
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {String}         Last quer
+ */
+export function getLastQuery( state, siteId ) {
```

Comment:
Considering we may switch to global selectors soon, I'd name this more explicitly.